plugins/techdocs/src/home/components/Tables/columns.tsx

```diff
@@ -30,11 +30,29 @@ function customTitle(entity: Entity): string {
  * @public
  */
 export const columnFactories = {
+  createTitleColumn(options?: { hidden?: boolean }): TableColumn<DocsTableRow> {
+    return {
+      title: 'Document',
```

Comment:
Is it intentional that `title` for `createTitleColumn` is the same as the `createNameColumn`? Both are 'Document'

Reply:
Mostly. It's meant to be a duplicate of `createNameColumn` but will generally be hidden